eth/downloader/peer.go

```diff
@@ -296,7 +297,7 @@ type peerSet struct {
 func newPeerSet() *peerSet {
 	return &peerSet{
 		peers: make(map[string]*peerConnection),
-		rates: msgrate.NewTrackers(log.New("proto", "eth")),
```

Comment:
looks like they were adding descriptors to each of these loggers? do we not need this?

Reply:
It's passed into the context of the logger, but it doesn't look like it's actually used anywhere.

Reply:
Looks like these were mainly used for tracking purpose, so we can look into it when we do the metrics, looks good for now